src/Uno.UI/UI/Xaml/FontFamily.wasm.cs

```diff
@@ -1,58 +1,48 @@
 ﻿#if __WASM__
+#nullable enable
+
 using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using Uno;
 
 namespace Windows.UI.Xaml.Media
 {
 	public partial class FontFamily
 	{
-		partial void Init(string fontName)
-		{
-			ParsedSource = ParseFontFamilySource(fontName);
-		}
+		private FontLoader _loader;
+
+		partial void Init(string fontName) => _loader = FontLoader.GetLoaderForFontFamily(this);
+
+		/// <summary>
+		/// Contains the font-face name to use in CSS.
+		/// </summary>
+		internal string CssFontName => _loader.CssFontName;
 
 		/// <summary>
-		/// Contains the parsed font family for use in WASM
-		/// (matches CSS @font-face's font-family)
+		/// Use this to launch the loading of a font before it is actually required to
+		/// minimize loading time and prevent potential flicking.
 		/// </summary>
-		internal string ParsedSource { get; private set; }
+		public static void Preload(FontFamily family) => family._loader.LaunchLoading();
```

Comment:
Also, could there be a "async" variant of the `Preload` method as well? So that the user could actually `await` its completion to be sure that all required fonts are ready - this could then be used in `OnLaunched` before the first page navigation.

Reply:
Done!